docs/cli/argo_resubmit.md

```diff
@@ -65,6 +65,7 @@ argo resubmit [WORKFLOW...] [flags]
   -s, --argo-server host:port          API server host:port. e.g. localhost:2746. Defaults to the ARGO_SERVER environment variable.
       --as string                      Username to impersonate for the operation
       --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
+      --as-uid string                  UID to impersonate for the operation
```

Comment:
Is this change intended in this PR?

Reply:
it is a consquence of the upgrade

Reply:
Anything that needs to be documented in upgrading guide?